### PR TITLE
To do feature created for To Do List Project

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**
Used webpack.
Made as one can add tasks.
Made as one can remove any task.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const taskValue = taskInput.value.trim();
+
+  const li = document.createElement('li');
+  li.innerHTML = `${taskValue} <button class="remove-button">X</button>`;
+  todoList.appendChild(li);
+
+  taskInput.value = '';
+  const removeButton = li.querySelector('.remove-button');
+  removeButton.addEventListener('click', removeTask);
+};
```
Comment (AbayomiOlaoye) [inline src/index.js:24]: - [ ] While you have made a brilliant feature functional such that it allows users to add to-do list items, I do not think this is a necessity for this milestone :clap:. A such you may need to check the requirement again to be sure of its expectations. According to the project requirement (shown below), you are expected to have an `array` of tasks as objects with some required `keys` pairs e.g. `description`, `completed`, and `index`. 

image
  - [ ] After this, it is expected that you create a function that traverses through or iterate the array in order to display each task according to your template and design, with much consideration for the `wireframe`/video in the project requirement.
  - [ ] And please, to make it really functional, do not forget to make the tasks get created in the UI when page loads according to their indexes/indices :wink:  
_Kindly implement these and submit them again for review_

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @SAZZAD1Q2 :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Nice User Interface styling :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Clean and DRY code :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  background-color: #bba4a4;
+  color: white;
+  border: none;
+  border-radius: 45%;
+  cursor: pointer;
+}
+
+li button:hover {
+  background-color: #c47c6dec;
+}
```
Comment (CollinsTatang) [inline src/style.css:54]: - [ ] Please style your project to match the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) design structure.
**Given Design**
fix
**Your Design**|

fix2

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi, @SAZZAD1Q2 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @SAZZAD1Q2 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Changes implemented :heavy_check_mark: 

## Optional suggestions

- [ ] **Please make sure the commit messages are meaningful and start with capital letter, also try to write them in imperative tense** :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**